### DISCUSSION TITLE
Railloader version

### DISCUSSION THREAD

**Discussion by hjp766** [Erabior/RouteManager#96] [Ideas]
Evening,

Reading Zamu's comments re some of the planned advanced mods mainly to do with CTC, any chance RouteManager could have a railloader version for compatibility?

**Comment by AdamC1228** [comment]
I'm sorry but I just don't follow what your saying here.

**Comment by Kaceyproductions** [comment]
[Railloader](https://railroader.stelltis.ch/) is a modloader dedicated to Railroader, made by Zamu. 

Adding my +1 to recommending porting to Railloader. RM is possibly the most popular mod currently, but it's also the single mod in most people's lists that requires a different loader.
